### DISCUSSION TITLE
Failed to get Db2 level: NOT AUTHORIZED TO COMMAND WRKPTFGRP

### DISCUSSION THREAD

**Discussion by buzzia2001** [codefori/vscode-ibmi#2014] [IBM i]
Hello, 
we are making some tests on a new lpar with a *PGMR user.
We have denied some commands like WRKPTFGRP because we don't want that a user can make actions on PTFGRP.
When we use db2 extension on vscode on output log i can see this message: Failed to get Db2 level: NOT AUTHORIZED TO COMMAND WRKPTFGRP 
If i check on QAUDJRN i'm able to find an AF for that command.
I only want to know what could appen in this situation without having the output of this command.
Thank you

**Comment by johnbodia** [comment]
I am using 2 different computers to connect, one was working the other one was not.  I removed (renamed) the .jar files from the ifs /home/myusername/.vscode directory and things started to work.  renamed both the codeforiserver and mapepire_server (this one was recreated when I logged in successfully).

**Comment by sebjulliand** [comment]
Same as this: https://github.com/codefori/vscode-ibmi/issues/2267
